nautobot/extras/context_managers.py

```diff
@@ -152,3 +159,57 @@ def web_request_context(
         for object_change in ObjectChange.objects.filter(request_id=change_context.change_id).iterator():
             enqueue_job_hooks(object_change)
             enqueue_webhooks(object_change)
+
+
+@contextmanager
+def object_changelogs_bulk_operation(objs, user, action):
```

Comment:
I'm not super happy with this function name but I'm having a hard time thinking of a better one. Maybe something like `deferred_change_logging_for_bulk_operation()`?